docs/index.md

```diff
@@ -66,3 +66,6 @@ DSP-TOOLS provides the following functionalities:
   The mapping has to be provided with a JSON file.
 - [`dsp-tools start-stack / stop-stack`](https://docs.dasch.swiss/latest/DSP-TOOLS/cli-commands/#start-stack)
   assist you in running a DSP stack on your local machine.
+- [`dsp-tools rosetta`](https://docs.dasch.swiss/latest/DSP-TOOLS/cli-commands/#rosetta)
```

Comment:
off topic, but because I'm seeing it right now: is there a reason for using absolute links here?

Reply:
yes. relative links wouldn't work on https://pypi.org/project/dsp-tools/

Reply:
makes sense. Doesn't that normally use the Readme? How did you override that behaviour?

Reply:
That's defined here: https://github.com/dasch-swiss/dsp-tools/blob/main/pyproject.toml#L8

For me, the readme and the pypi landing page have two different goals:
 - readme: teach a developer how to get started if he wants to code
 - pypi: teach a user how he can use the package

That's why I take docs/index.md: For me, it has the same purpose than the pypi landing page

Reply:
yes, makes sense.